style.css

```diff
@@ -0,0 +1,370 @@
+}
+
+.projectBox h4{
+    font-size: 48px;
+    letter-spacing: 5px;
+    font-weight: bold;
+    margin-top: 25%;
+    color: #FFFFFF;
+}
+
+.projectBox P{
+    font-size: 24px;
+    letter-spacing: 2px;
+    font-weight: bold;
+    color: #FFFFFF;
+}
+
+.projectBox:hover{
+    opacity:1;
+}
+
+.projectBox a:visited{
+    color: #FFFFFF;
+    text-decoration: none;
+}
+
+.project1{
+    background-image: url(images/pilotshop.png);
+    background-size: cover;
+    background-position: center;
+}
+
+.project2{
+    background-image: url(images/maydenlogo.png);
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+}
+
+.project3{
+    background-image: url(images/workprogress1.jpg);
+    background-size: cover;
+    background-position: center;
+}
+
+.project4{
+    background-image: url(images/workprogress2.jpg);
+    background-size: cover;
+    background-position: center;
+}
+.project5{
+    background-image: url(images/workprogress3.jpg);
+    background-size: cover;
+    background-position: center;
+}
+.project6{
+    background-image: url(images/workprogress4.jpg);
+    background-size: cover;
+    background-position: center;
+}
+
+.boxes{
+    overflow: auto;
+}
+
+/*ABOUT ME*/
+
+.aboutMe{
+    text-align: center;
+    overflow: auto;
+}
+
+.aboutMe h2{
+    font-size: 48px;
+    color: #565254;
+    font-weight: normal;
+    letter-spacing: 6px;
+}
+
+.aboutMe p{
+    letter-spacing: 2px;
+    line-height: 18px;
+    font-size: 16px;
+    color: #565254;
+}
+
+.aboutMe .archWorks{
+    font-size: 24px;
+}
+
+.archBox{
+    width: 33.3%;
+    height: 250px;
+    float: left;
+    border: 1px solid black;
+}
+
+/*CONTACT ME*/
+
+footer{
+    text-align: center;
+    background-color: #272727;
+    width: 100%;
+}
+
+footer button{
+    border: 2px solid #14a76c;
+    margin-bottom: 20px;
+    padding: 10px;
+    background-color: transparent;
+    color: #14a76c;
+    font-size: 18px;
+}
+
+footer .contact{
+    padding-top: 5px;
+    text-align: center;
+    color: #747474;
+    margin: 0px;
+}
+
+footer a{
+    text-decoration: none;
+    color: #14a76c;
+}
+
+footer a:visited{
+    color: #14a76c;
+}
+
+.logoContainer a img{
+    height: 50px;
+    width: auto;
+    opacity: 0.8;
+    margin-left: 20px;
+}
+
+.logoContainer{
+    float: right;
+    display: inline-block;
+    margin-top: -70px;
+}
+
+.copyright{
+    color: #FFFFFF;
+    float: left;
+    margin-top: -40px;
+    font-weight: lighter;
+    opacity: 0.8;
+}
+
+/*MEDIA QUERIES*/
+
+@media only screen and (max-width:500px) {
+    .boxes .projectBox{
+        width: 100%;
+        height: 300px;
+    }
+    .navBar li{
+        font-size: 14px;
+        padding: 0;
+        margin: 30px 10px;
+        border: none;
+    }
+
+    .navBar li a{
+        margin: 10px;
+    }
+
+    .navBar a:hover{
+        border: none;
+    }
+
+   footer{
+       height: 300px}
+}
+
+
+@media only screen and (max-width:768px) {
```

Comment:
Before this query happens, your copyright part of the website overlap with the button around 850px

Reply:
doneeeeee